samples/nrf_compress/mcuboot_update/README.rst

```diff
@@ -16,58 +16,116 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
-Note the following additional requirements:
+Overview
+********
 
-* MCUboot configured in upgrade-only mode
-* Single update image
+This sample is using the :ref:`nrf_compression` library.
+
+When this sample is built with the :ref:`mandatory Kconfig options <nrf_compression_mcuboot_compressed_update_setup>`, the build system will automatically generate the update binary files with compressed image data and flags set.
+These files match the :ref:`requirements for MCUboot image compression <mcuboot_image_compression_setup>`.
+
+Configuration
+*************
+
+|config|
+
+.. _nrf_compression_mcuboot_compressed_update_setup:
+
+Setup
+=====
+
+Before you can compile this sample, you need to set up the following requirements:
+
+* Configure MCUboot in the upgrade-only mode by setting the Kconfig option :kconfig:option:`CONFIG_BOOT_UPGRADE_ONLY` to ``y``.
+* Configure single-update image by setting the Kconfig option :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER` to ``1``.
```

Comment:
these are done automatically, users do not do anything with these

Reply:
Roger, repushed.